src/clarity-angular/popover/popover.directive.ts

```diff
@@ -47,14 +55,23 @@ export class PopoverDirective {
         //inside of the popover. For Eg: Button Groups
         embeddedViewRef.detectChanges();
 
-        this._popoverInstance = new Popover(embeddedViewRef.rootNodes[0]);
-        this._popoverInstance.anchor(
-            this.anchorElem, this.anchorPoint, this.popoverPoint, this.popoverOptions);
+        // filter out other nodes in the view ref so we are only left with element nodes
+        let elementNodes: HTMLElement[] = embeddedViewRef.rootNodes.filter((node: any) => {
+           return node.nodeType === 1;
+        });
```

Comment:
So `<ng-container *clrPopover="true">Hello World</ng-container>` doesn't work? I'm okay with that, I just want to make sure I understand the constraints.